pkg/cmd/validate.go

```diff
@@ -83,26 +84,28 @@ func NewRootCommand() *cobra.Command {
 		RunE:  invoked.Run,
 	}
 	res.Flags().StringVarP(&invoked.version, "version", "", "", "Kubernetes version to validate native resources against. Required if not connected directly to cluster")
-	res.Flags().StringVarP(&invoked.localFilesDir, "local-schemas", "l", "", "--local-schemas=./path/to/schemas/dir. Path to a directory with format: /apis/<group>/<version>.json for each group-version's schema.")
+	res.Flags().StringVarP(&invoked.localSchemasDir, "local-schemas", "l", "", "--local-schemas=./path/to/schemas/dir. Path to a directory with format: /apis/<group>/<version>.json for each group-version's schema.")
+	res.Flags().StringVarP(&invoked.localCRDsDir, "local-crds", "", "", "--local-crds=./path/to/crds/dir. Path to a directory containing .yaml or .yml files for CRD definitions.")
```

Comment:
Looks like `local-schemas` is holding onto the shorthand here. I am not opinionated about the shorthands used. Opening up discussion for whether `l` should be used at all, or proposals for new shorthands for `local-schemas` and `local-crds`

Reply:
`-s` for schemas and `-c` for CRDs ?

Reply:
Switched to `-s` for schemas and `-c` for CRDs, WDYT ?

Reply:
Im thinking since it is still early in the project we leave these flags without a shorthand for now. Can always add them later, but once they are set, they are set.

Reply:
Done 👍